ndb/tests/unit/test_model.py

```diff
@@ -2722,6 +2656,55 @@ def test__put():
         with pytest.raises(NotImplementedError):
             entity._put()
 
+    @staticmethod
+    def test__lookup_model():
+        class ThisKind(model.Model):
+            pass
+
+        sentinel = object()
+        assert model.Model._lookup_model("ThisKind")
+        assert model.Model._lookup_model("NoKind", sentinel) is sentinel
+
+        with pytest.raises(model.KindError):
+            model.Model._lookup_model("NoKind")
```

Comment:
Should this unit test not be split into 3?

Reply:
2b1f9b5